pkg/virtualtable/virtualtable_test.go

```diff
@@ -32,12 +32,12 @@ func Test_AddGetVTables(t *testing.T) {
 	config.InitializeDefaultConfig()
 	_ = InitVTable()
 
-	vTableBaseDir = "vtabbase/"
+	VTableBaseDir = "vtabbase/"
 	VTableMappingsDir = "vtabbase/mappings/"
 	VTableTemplatesDir = "vtabbase/templates/"
-	vTableAliasesDir = "vtabbase/aliases/"
+	VTableAliasesDir = "vtabbase/aliases/"
```

Comment:
these can be vTable* instead of VTable* ,  since it is not being used in any other pkg